Solutions/SharpArch.NHibernate/SessionFactoryAttribute.cs

```diff
@@ -9,7 +9,7 @@
     ///     communicate with the database.  This allows you to declare different repositories to 
     ///     communicate with different databases.
     /// </summary>
-    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
```

Comment:
This is default value,  no need to duplicate it it code.

Regards, Vitaliy
On Nov 25, 2015 7:07 PM, "Seif Attar" <notifications@github.com> wrote:

> In Solutions/SharpArch.NHibernate/SessionFactoryAttribute.cs
> <https://github.com/sharparchitecture/Sharp-Architecture/pull/125#discussion_r45933151>
> :
>
> > @@ -9,7 +9,7 @@
> >      ///     communicate with the database.  This allows you to declare different repositories to
> >      ///     communicate with different databases.
> >      /// </summary>
> > -    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
>
> @shalt <https://github.com/shalt> why this change?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sharparchitecture/Sharp-Architecture/pull/125/files#r45933151>
> .
>